text/setup.py

```diff
@@ -18,10 +18,10 @@
 submodule = 'text'
 install_requires = [
     # version ranges added in ag.get_dependency_version_ranges()
-    'numpy',
-    'scipy',
-    'pandas',
-    'scikit-learn',
+    'numpy<=1.22',
+    'scipy<1.8.0',
+    'pandas<1.4',
+    'scikit-learn<1.1',
```

Comment:
For version cap, the general rule is to make the supported version strictly smaller than the next minor version,e.g. xxx<1.2. For common packages, these are covered in https://github.com/awslabs/autogluon/blob/master/core/src/autogluon/core/_setup_utils.py#L16

Reply:
Both minimum and maximum are capped now.